Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -55,6 +56,12 @@ public int LaunchFile(string file, bool debug)
         {
             var nodePath = GetNodePath();
 
+            if(NpmInstallCommand.IsInstalling)
+            {
+                Nodejs.ShowNpmIsInstalling();
+                return VSConstants.E_ABORT;
```

Comment:
It looks like the error case below returns `S_OK`.

Reply:
Changed.